src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/InvalidHwFCallbackHandler.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.INVALID_HWF_REFERENCE;
+
+@Service
+@RequiredArgsConstructor
+public class InvalidHwFCallbackHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = List.of(INVALID_HWF_REFERENCE);
+    private final ObjectMapper objectMapper;
+
+    private final Map<String, Callback> callbackMap = Map.of(
+        callbackKey(ABOUT_TO_START), this::emptyCallbackResponse,
+        callbackKey(ABOUT_TO_SUBMIT), this::aboutToSubmit);
```

Comment:
Please add the following for the SUBMIT part
callbackKey(SUBMITTED), this::emptySubmittedCallbackResponse